### PR TITLE
Update telemetry.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated plugin config information (https://github.com/mattermost/mattermost-server/pull/14846) and minor formatting fixes.

@amyblais, apologies for this not being on the doc branch.

### CODE REVIEW

File: source/administration/telemetry.rst [hunk 1]
```diff
@@ -112,15 +112,15 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
...
   - *Core Feature Discovery Diagnostics:* Created, edited or deleted a message; posted a message containing a hashtag, link, mention or file attachment; searched for a term; searched for flagged posts or recent mentions
   - *Advanced Feature Discovery Diagnostics:* Reacted to a message; favorited or un-favorited a channel; flagged or un-flagged a message; pinned or un-pinned a message; replied to a message; expanded the right-hand sidebar; started or finished a WebRTC video call (only in v5.5 and earlier); created or deleted a personal access token; added or removed post:all or post:channels permission
   - *Integration Discovery Diagnostics:* Created or triggered a webhook or slash command; created, authroized or deleted an OAuth 2.0 app; created, posted, or deleted a custom emoji
   - *Plugin Discovery Diagnostics:* Number of installed plugins containing either server or webapp portions, or both; number of those plugins being activated
   - *Plugin Marketplace Diagnostics:* Plugin id, current version, and target version for all install and update events. Only sent when the default Marketplace is configured
```
Comment (justinegeffen) [inline source/administration/telemetry.rst:121]: @hanzei, does this need to be updated for https://github.com/mattermost/mattermost-server/pull/14846 as well?

File: source/administration/telemetry.rst [hunk 2]
```diff
@@ -79,19 +79,19 @@ Channel Moderation Configuration Information (Enterprise Edition Only)
...
+  Information related to AD/LDAP groups, including number of groups synced to Mattermost, teams and channels associated to groups, teams and channels synced with groups, and number of group members.
 
 Plugin Configuration Information
-  Basic information including number of active and inactive plugins, which are using webapp or backend portions, and which `whitelisted Mattermost plugins <https://github.com/mattermost/mattermost-server/blob/master/app/diagnostics.go#L668>`_ are enabled along with their versions.  Some plugins may send summary data such as number of authenticated users of the plugin. 
+  Basic information including number of active and inactive plugins, which are using webapp or backend portions, and which `Mattermost plugins <https://github.com/mattermost/mattermost-server/blob/master/app/diagnostics.go#L668>`_ are enabled along with their versions. The list of plugins is obtained from the Marketplace. If the Marketplace can't be reached, the list of known plugins is used instead. Some plugins may send summary data such as number of authenticated users of the plugin.
```
Comment (hanzei) [inline source/administration/telemetry.rst:85]: Should we mention here that the behaviour changed in 5.26?

File: source/administration/telemetry.rst (hunk 1 above)
Comment (hanzei) [reply source/administration/telemetry.rst:121]: These events are triggered in the Marketplace modal and hence can only happen for Marketplace plugins.

File: source/administration/telemetry.rst (hunk 2 above)
Comment (justinegeffen) [inline source/administration/telemetry.rst:85]:
```suggestion
  Basic information including number of active and inactive plugins, which are using webapp or backend portions, and which `Mattermost plugins <https://github.com/mattermost/mattermost-server/blob/master/app/diagnostics.go#L668>`_ are enabled along with their versions. Some plugins may send summary data such as number of authenticated users of the plugin. From version 5.26, the list of plugins is obtained from the Marketplace. If the Marketplace can't be reached, the list of known plugins is used instead.
```

Comment (justinegeffen) [reply source/administration/telemetry.rst:85]: Good call, @hanzei. I've added that and reorganized the sentence so it's obvious what the change is. LMK what you think.

Comment (hanzei) [reply source/administration/telemetry.rst:85]: :+1: from my side

**Pull request merged**